xbmc/guilib/DirtyRegion.h

```diff
@@ -35,3 +35,23 @@ class CDirtyRegion : public CRect
 };
 
 typedef std::vector<CDirtyRegion> CDirtyRegionList;
+
+class CGUIRenderInfo
+{
+public:
+  CGUIRenderInfo() { Reset(0); }
+  void Reset(unsigned int t) { m_currentTime = t; m_dirtyregions.clear(); m_visible = 0; m_total = 0; }
+  void IncVisible() { m_visible++; }
+  void IncTotal() { m_total++; }
+  int GetVisible() { return m_visible; }
+  int GetTotal() { return m_total; }
+  unsigned int GetTime() { return m_currentTime; }
```

Comment:
The getters should be const. For example <code>unsigned int GetTime() const</code>.